aopy/analysis/base.py

```diff
@@ -602,6 +602,61 @@ def calc_stat_over_dist_from_pos(elec_data, elec_pos, pos, statistic='mean', bin
 
     return dist, stat
 
+def calc_stat_over_angle_from_pos(elec_data, elec_pos, origin, statistic='mean', bins=20):
+    '''
+    For spatial data, calculate a statistic against the angle between each position and a given origin.
+
```

Comment:
Overall this function makes sense. I think the description could be more informative about the order of operations. For example, specify that it bins all data based on angle, then for data at the same angle it computes a statistic.